op-node/p2p/sync.go

```diff
@@ -608,8 +610,8 @@ func (s *SyncClient) doRequest(ctx context.Context, id peer.ID, expectedBlockNum
 		return fmt.Errorf("failed to read version part of response: %w", err)
 	}
 	version := binary.LittleEndian.Uint32(versionData[:])
-	if version != 0 {
-		return fmt.Errorf("unrecognized ExecutionPayload version: %d", version)
+	if version != 0 && version != 1 {
```

Comment:
Should this be an OR?

Reply:
Wouldn't an OR  always evaluate to true - if version is 0 it's != 1 and if it's 1 it's != 0 so one branch would always be true.  I am notoriously bad at boolean logic though...